AUTHORS.rst

```diff
@@ -0,0 +1,30 @@
+..
+    This file is part of DoSchema.
+    Copyright (C) 2016 CERN.
+
+    DoSchema is free software; you can redistribute it
+    and/or modify it under the terms of the GNU General Public License as
+    published by the Free Software Foundation; either version 2 of the
+    License, or (at your option) any later version.
+
+    DoSchema is distributed in the hope that it will be
+    useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with DoSchema; if not, write to the
+    Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+    MA 02111-1307, USA.
+
+    In applying this license, CERN does not
+    waive the privileges and immunities granted to it by virtue of its status
+    as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+
+Authors
+=======
+
+JSON Schema utility functions and commands.
+
+- CERN <info@invenio-software.org>
```

Comment:
Ah yeah you are right. Thanks for the reminder. It will be fixed in a later commit.